lib/splitio.dart

```diff
@@ -92,6 +94,40 @@ class Splitio {
     return client;
   }
 
+  Future<List<String>> splitNames() async {
```

Comment:
Also check, if it is okey to have this methods at the "sdk" level, rather than in a "manager" level. 
In other way, if it should be called like `splitio.splitNames();` or `splitio.manager().splitNames();`.